group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorService.java

```diff
@@ -935,6 +955,78 @@ public CompletableFuture<OffsetFetchResponseData.OffsetFetchResponseGroup> fetch
         }
     }
 
+    /**
+     * See {@link GroupCoordinator#describeShareGroupOffsets(RequestContext, DescribeShareGroupOffsetsRequestData)}.
+     */
+    @Override
+    public CompletableFuture<DescribeShareGroupOffsetsResponseData> describeShareGroupOffsets(
+        RequestContext context,
+        DescribeShareGroupOffsetsRequestData requestData
+    ) {
+        if (!isActive.get()) {
```

Comment:
I think you should move the impl for this method to OffsetsManager in line with existing methods.